google-summer-of-code/GSOC-STUDENT-TEMPLATE.md

```diff
@@ -39,8 +43,8 @@
 
 ### About me
 
-> Add your name, personal details and contact information (Slack ID, E-Mail). Optionally, add why you would you consider yourself as the best person for this project.
+> Introduce yourself here in a brief manner. Try to answer basic qustions like Who are you? What are your interest?. Add your name, personal details and contact information (Slack ID, E-Mail).
```

Comment:
`Introduce yourself here in a concise manner. Try to answer basic questions like Who are you? What are your interests? Add your name, personal details and contact information (Slack ID, E-Mail).`

Reply:
ok